bundles/org.openhab.binding.shelly/README.md

```diff
@@ -80,35 +80,35 @@ See section [Discovery](#discovery) for details.
 
 ### Shelly Plus series (Generation 2+3+4)
 
-| thing-type           | Model                                                    | Vendor ID                                   |
-|----------------------|----------------------------------------------------------|---------------------------------------------|
-| shellyplus1          | Shelly Plus 1 with 1x relay                              | SNSW-001X16EU, S3SW-001X16EU, S4SW-001X16EU |
-| shellyplus1pm        | Shelly Plus 1PM with 1x relay + power meter              | SNSW-001P16EU, S3SW-001P16EU, S4SW-001P16EU |
-| shellyplus2pm-relay  | Shelly Plus 2PM with 2x relay + power meter, relay mode  | SNSW-002P16EU, SNSW-102P16EU                |
-| shellyplus2pm-roller | Shelly Plus 2PM with 2x relay + power meter, roller mode | SNSW-002P16EU, SNSW-102P16EU                |
-| shellyplusplug       | Shelly Plug-S Gen 3, Shelly AZ Plug                      | SNPL-00112EU                                |
-| shellyplusplug       | Shelly Outdoor Plug-S                                    | S3PL-20112EU                                |
```

Comment:
Why did you remove this line? It was added in #18814.

Reply:
re-added